Frontend/src/main/java/com/harmonia/client/UserClient.java

```diff
@@ -44,12 +68,11 @@ public ResponseEntity<Void> removeUser(int userId) {
         return restTemplate.exchange(USERS_DELETE_URL, HttpMethod.DELETE, request, Void.class, urlParameters);
     }
 
-    public UserPO editPassword(int target, String newpassword) {
```

Comment:
Is this one old method that we are not using anymore?

Reply:
there was two methods for editPassword. This one was missing the urlParameters so it was doing nothing. That is why it is now deleted.